core/storage/rocksdb/rocksdb.cpp

```diff
@@ -115,6 +115,23 @@ namespace kagome::storage {
     return space_ptr;
   }
 
+  void RocksDb::dropColumn(kagome::storage::Space space) {
+    auto space_name = spaceName(space);
+    auto column_it =
+        std::find_if(cf_handles_.begin(),
+                     cf_handles_.end(),
+                     [&space_name](const ColumnFamilyHandle &handle) {
+                       return handle.name == space_name;
+                     });
+    if (cf_handles_.end() == column_it) {
+      throw DatabaseError::INVALID_ARGUMENT;
+    }
+    auto &column = *column_it;
+    db_->DropColumnFamily(column.handle);
+    db_->DestroyColumnFamilyHandle(column.handle);
+    db_->CreateColumnFamily({}, column.name, &column.handle);
```

Comment:
Does constructor set "create missing column family = true"?
(In case we drop and don't create)

Do we need to check returned status/error here (and throw or outcome)?

Reply:
1. Constructor is set to true. Nevertheless, I see no reason to omit column initialization here.
2. Will fix it in upcoming commit.